src/Microsoft.Diagnostics.Monitoring.RestServer/Controllers/DiagController.cs

```diff
@@ -139,6 +139,7 @@ public Task<ActionResult<IEnumerable<Models.ProcessIdentifier>>> GetProcesses()
         // FileResult is the closest representation of the output so that the OpenAPI document correctly
         // describes the result as a binary file.
         [ProducesResponseType(typeof(FileResult), StatusCodes.Status200OK)]
+        [RequestLimit(MaxConcurrency = 1)]
```

Comment:
For each action that has this RequestLimit attribute, you should add [ProducesResponseType(typeof(void), StatusCodes.Status429TooManyRequests)] so that the OpenAPI document reflects the HTTP status code.